src/app/map/map.component.ts

```diff
@@ -185,14 +202,15 @@ export class MapComponent implements AfterViewInit {
     const colorRe = /(\d+), (\d+), (\d+)/;
     const layer = new GeoJsonLayer({
       id: LAYER_ID,
-      data: this._features,
+      data: this._page === 0 ? this._features : [this._features[this._page - 1]],
       pickable: true,
       autoHighlight: true,
       highlightColor: [219, 68, 55], // #DB4437
       stroked: this.hasStroke(),
       filled: true,
       extruded: false,
       elevationScale: 0,
+      binary: true,
```

Comment:
What does this do?

Reply:
Oh, good catch! I was just playing around with some of the functionality in the maps API. I will revert it.